app/routes.py

```diff
@@ -1,2 +1,61 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, has_moon=None):
```

Comment:
for `has_moon` you can use a boolean value directly as the default. So in this case `has_moon=False`. We initially used `None` for the inventory list in swap meet because we wanted to tell whether the call had passed in the inventory list.